core/telemetry.go

```diff
@@ -71,6 +71,11 @@ func AroundFunc(
 		attribute.String(telemetry.DagCallAttr, callAttr),
 	}
 
+	// HACK: probably some more elegant way to do make Module.call passthrough
+	// if base == "Module" && id.Field() == "call" {
+	// 	attrs = append(attrs, attribute.Bool(telemetry.UIPassthroughAttr, true))
+	// }
```

Comment:
noting to cleanup before merge (not sure if it should be left in and cleaned up or just rm'd)